CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+## General
+
+While this project is in the OCaml Labs organization, contributions from others
+are very much welcome. We're trying to build a tool that works for many users.
+
+If you're unclear whether a feature would fit the scope of `dune-release` don't
+hesitate to open an issue to gauge the interest.
+
+## Setting up your working environment
+
+If you want to contribute to the project you'll first need to install the dependencies.
+You can do it via `opam`:
+
+```sh
+$ git clone git@github.com:ocamllabs/dune-release.git
+$ cd dune-release
+$ opam switch create ./ ocaml-base-compiler.4.13.1 --no-install
```

Comment:
I think this can be abreviated `opam switch create ./ --deps-only -t`, let me check !

Reply:
That's a good point, I wasn't aware this is possible. I had to use `opam switch create ./ ocaml-base-compiler.4.13.1 --deps-only -t`, otherwise the switch wouldn't have a compiler invariant and that seemed somewhat bad to me.

Reply:
I think it's alright not to have an invariant here but we can always change that later!